crates/matrix-sdk-base/src/media.rs

```diff
@@ -86,211 +76,93 @@ impl UniqueKey for MediaType {
 /// A request for media data.
 #[derive(Clone, Debug)]
 pub struct MediaRequest {
-    /// The type of the media file.
-    pub media_type: MediaType,
+    /// The source of the media file.
+    pub src: MediaSource,
```

Comment:
Can we call this and the accompanying methods `source` instead, I prefer plain English, unless things become too long.

Reply:
Sure, though it would be nice to do the same in Ruma then, for consistency. cc @zecakeh wdyt?

Reply:
Sure, I don't have a strong opinion about this